scripts/split-paired-reads.py

```diff
@@ -57,8 +61,15 @@ def main():
     filenames = [infile]
     check_space(filenames, args.force)
 
-    out1 = os.path.basename(infile) + '.1'
-    out2 = os.path.basename(infile) + '.2'
+    if args.output_directory:
+        if not os.path.exists(args.output_directory):
+            os.makedirs(args.output_directory)
+        out1 = args.output_directory + '/' + os.path.basename(infile) + '.1'
```

Comment:
maybe use os.path.join here? not sure we'll ever support Windows, admittedly, but.